CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## v1210
+
+* Staging release date: TBD
+* Production APM-injected release date: TBD
+* Production Standalone release date: TBD
+
+
+### PageHide events are now accounted for during page unload events
+`PageHide` events are used to query CLS values, which led to inconsistencies if the `pageHide` value was never set.  Now when `pageUnload` fires, if `pageHide` has not already been set, it will set at to the time of unload. 
```

Comment:
"`pageHide` value was never set" really means that PageViewTiming event with that type was not collected. It might be better to say that or maybe something like "... if the page was never hidden".